content/authors/laura-graham/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Laura J. Graham
```

Comment:
In order to display author name correctly in the blog post (see [preview](https://deploy-preview-490--ropensci.netlify.com/blog/2019/07/02/grainchanger/)) I think `name` in author file has to precisely match authors folder name, and `name` in blog post YAML.

Solutions, depending on whether you want to retain your "J."
1) in `_index.md` if you use `name: Laura Graham` , it should display your name as "Laura Graham" in the blog post preview

Or 2) to display as "Laura J. Graham", change `content/authors/laura-graham/_index.md` to `content/authors/laura-j.-graham/_index.md` and in YAML of blog post change 
```
authors:
  - Laura Graham
```

to 
```
authors:
  - Laura J. Graham
```

Reply:
Here's a middle-initial example

- https://raw.githubusercontent.com/ropensci/roweb2/master/content/authors/anik%C3%B3-b.-t%C3%B3th/_index.md
- https://ropensci.org/authors/anik%C3%B3-b.-t%C3%B3th/
- https://raw.githubusercontent.com/ropensci/roweb2/master/content/blog/2017-11-28-ropensci-changes.md
- https://ropensci.org/blog/2017/11/28/ropensci-changes/